### ISSUE TITLE
Passthrough adapter to ESXi VM not working

### ISSUE THREAD

**Issue by sjkjs** [magic-blue-smoke/Dual-Edge-TPU-Adapter#37]
I'm running ESXi 8.0.1 on an HP DL380p Gen8 and want to pass through the Coral devices to a VM.

If I SSH into the ESXi server and run lspci, it shows:

`0000:0c:00.0 Non-VGA unclassified device: Global Unichip Corp. Coral Edge TPU
0000:0d:00.0 Non-VGA unclassified device: Global Unichip Corp. Coral Edge TPU`

Under ESXi PCI devices I see that passthrough is not supported:

![esxi](https://github.com/magic-blue-smoke/Dual-Edge-TPU-Adapter/assets/2536157/9f5f0017-9130-4fff-a0b6-c7c2e9aab4cc)

Through vCenter it says "This device cannot be made available for VMs to use" when selecting it.

![vcenter](https://github.com/magic-blue-smoke/Dual-Edge-TPU-Adapter/assets/2536157/14019614-b1ca-4570-bd4f-24b2bc25626e)

If I click toggle passthrough, it displays an error saying "This device cannot be made available for VMs to use"

Has anyone gotten this to work before?

**Comment by jahegu** [comment]
same problem, any new?

**Comment by magic-blue-smoke** [comment]
Passthrough requires VM-d/iommu support and VM-d/iommu enabled in BIOS. Is it something what was already tried?

**Comment by sjkjs** [comment]
Yes, VT-d was enabled and other devices are successfully being used in passthrough mode.

I've since given up and sold the Coral so I can't test any further.
